ansible/roles/images/tasks/load.yaml

```
@@ -9,12 +9,22 @@
   register: images_cache_find_result
   changed_when: false
 
-- name: import images from files in the local images cache  # noqa no-changed-when
-  shell:
-    executable: /bin/bash
-    cmd: ctr --address {{ containerd_cri_socket }} --namespace k8s.io images import <(gzip --decompress --stdout {{ item }})
+- block:
+    - name: import images from files in the local images cache # noqa no-changed-when
+      shell:
+        executable: /bin/bash
+        cmd: |
+          ctr --address {{ containerd_cri_socket }} --namespace k8s.io images import <(gzip --decompress --stdout {{ item }});
+          if [[ $? -ne 0 ]]; then
+            {{ mindthegap_binary_location_on_remote }} import image-bundle --image-bundle={{ item }};
```

Can we flip the statements, `mindthegap` is where we are moving towards.
The way its there currently, we would be decompressing every `item` before failing.
```suggestion
          ctr --address {{ containerd_cri_socket }} --namespace k8s.io images import <(gzip --decompress --stdout {{ item }});
          if [[ $? -ne 0 ]]; then
            {{ mindthegap_binary_location_on_remote }} import image-bundle --image-bundle={{ item }};
```

---

sure we can flip it